notebooks/community/pipelines/google_cloud_pipeline_template.ipynb

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
<p>Seems hanging. After reusing the pipeline template, I suggest something to verify it happened. Like get the model/endpoint ID.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>